src/main/resources/languages/en/messages.yml

```diff
@@ -136,7 +136,7 @@ guide:
     description:
     - '&7Slimefun is an open-source project'
     - '&7and maintained by a large community of people.'
-    - '&7Over &e%contributors% &7people have worked on'
+    - '&7Many people have worked on'
```

Comment:
Iirc this won't update for servers automatically so instead it'll have a not filled placeholder

Reply:
for english it will automaticall update.
for other languages it need to be manually changed through crowdin.

Reply:
Isn't that only for new keys? It won't override it afaik

Reply:
Otherwise the point of configs would be kinda moot